src/components/shared/Tooltip.css

```diff
@@ -47,6 +47,14 @@
   margin-right: 3px;
 }
 
+.tooltipIcon {
+  width: 14px;
+  height: 14px;
+  display: inline-block;
+  margin-right: 3px;
+  align-self: flex-end;
+}
```

Comment:
Yeah, that doesn't look very good.

But, (ignoring the fact that the icon is too close to the text in the image), putting it on the right side of the text looks a bit off to me:
![screenshot from 2018-05-19 23-06-17](https://user-images.githubusercontent.com/1911402/40273280-bf93df88-5bbd-11e8-91bd-4c7913a7d990.png)
I think the favicon goes better together with the domain name of the URL rather than the path on the right side. This exacerbates as the URL gets longer.

Perhaps we can align the URL with the function name above it, then it looks less bad if there's no favicon. What do you think?
![screenshot from 2018-05-19 23-28-09](https://user-images.githubusercontent.com/1911402/40273332-c5370824-5bbe-11e8-9528-e60351dc5c3c.png)

Reply:
> Perhaps we can align the URL with the function name above it, then it looks less bad if there's no favicon. What do you think?

This looks pretty good to me and seems like the best option in my opinion.

I don't like the proposal of putting it on the right, probably because favicons are on the left in all other contexts where they're used.

Reply:
OK, if @mstange :+1: I'm OK too :)

Reply:
Ok, so I pushed new changes that implements the suggested option.
I'm not super pleased with that approach however, mainly because of how it looks when there's no lib/URL at all in the tooltip:
![screenshot from 2018-05-26 21-43-59](https://user-images.githubusercontent.com/1911402/40579729-90d301f8-612e-11e8-9443-4b0cbfc46dbf.png)

@julienw 's suggestion to put the whole thing on the top right would solve that problem, but the tooltips tend to be very long if we need to squeeze both function name and URL on the same line.

I'm out of ideas on how to solve this without having text jumping.

Reply:
I don't understand - why is there an empty row if neither the favicon nor the url are present?

Reply:
I was going to answer that it's because the icon has a min-height, but now I realize that I should just add back the conditional rendering of the icon depending on whether `displayData.icon` is defined. I had removed it, believing it was unnecessary.

Thank you for asking! This shouldn't be problem anymore :)